src/Raven.Server/Documents/Indexes/Persistence/Lucene/LuceneSuggestionIndexWriter.cs

```diff
@@ -263,7 +263,7 @@ private void DisposeIndexWriter(bool waitForMerges = true)
 
             try
             {
-                writer?.Analyzer.Close();
```

Comment:
Can this have been concurrently changed here?

Reply:
If that code runs concurrently, we have a race condition at

            if (_indexWriter == null)
                 return;

So the assumption is that it doesn't